Makefile

```diff
@@ -142,11 +142,12 @@ dist: pipenv fetch-tools
 	-[ -n "$$NOSWITCH" ] || git checkout -
 
 fetch-tools:
-	-rm -Rf testsuite-tools/
-	-curl $(fetch_tools) | tar -xz
+	-rm -Rf ext/testsuite-tools/
+	-mkdir -p ext/
+	-curl $(fetch_tools) | tar -C ext/ -xz
 
 tools:
```

Comment:
I would add dependency on `ext/testsuite-tools/run.sh` or `fetch-tools` so it does not fail with file not found?

Reply:
No. `fetch-tools` is build action (called during container build) while `tools` is runtime action (as tools url doesn't have to be available at runtime), they are intentionally split